### PR TITLE
fix: Make packages/medusa/src/services/product-variant.js pass linting

### LINKED ISSUE #518: chore: Make packages/medusa/src/services/product-variant.js pass linting
In an effort to improve our code quality we want to have make sure that all files in the repo pass linting.

Please see this guide for a step-by-step guide for how to complete this task: https://www.notion.so/medusajs/Lint-challenge-checklist-b4efc761cb36499abf4726cad0dd02e4

🚀

### PR DESCRIPTION
**Pull request by Mihir50**
Make changes to the `product-variant.js` file to fix all the linting errors.
The file is passing `yarn lint` **successfully**.
Fixes #518

### CODE REVIEW

**Comment by Mihir50** [comment]
@olivermrbl @srindom  Can you please review this? Because today is the last day for hacktoberfest ;)

File: packages/medusa/src/services/product-variant.js [hunk 1]
```diff
@@ -246,7 +246,7 @@ class ProductVariantService extends BaseService {
    * Updates a variant.
    * Price updates should use dedicated methods.
    * The function will throw, if price updates are attempted.
-   * @param {string | ProductVariant} variant - the id of the variant. Must be a
+   * @param {string | ProductVariant} variantOrVariantId - the id of the variant. Must be a
    *   string that can be casted to an ObjectId
```
Comment (olivermrbl) [inline packages/medusa/src/services/product-variant.js:250]: Change to:
```
@param {string | ProductVariant} variantOrVariantId - variant or id of a variant.
```

File: packages/medusa/src/services/product-variant.js [hunk 2]
```diff
@@ -69,6 +67,7 @@ class ProductVariantService extends BaseService {
   /**
    * Gets a product variant by id.
    * @param {string} variantId - the id of the product to get.
+   * @param {Object} config - the config of the product to retrieve.
```
Comment (olivermrbl) [inline packages/medusa/src/services/product-variant.js:70]: Change to:
```
@param {Object} config - query config object for variant retrieval
```

File: packages/medusa/src/services/product-variant.js [hunk 3]
```diff
@@ -91,7 +90,8 @@ class ProductVariantService extends BaseService {
 
   /**
    * Gets a product variant by id.
-   * @param {string} variantId - the id of the product to get.
+   * @param {string} sku - The unique stock keeping unit used to identify the product variant.
+   * @param {Object} config the config of the product to retrieve.
```
Comment (olivermrbl) [inline packages/medusa/src/services/product-variant.js:94]: Change to:
```
@param {Object} config - query config object for variant retrieval
```

File: packages/medusa/src/services/product-variant.js [hunk 4]
```diff
@@ -533,6 +531,7 @@ class ProductVariantService extends BaseService {
 
   /**
    * @param {Object} selector - the query object for find
+   * @param {Object} config the config object for find
```
Comment (olivermrbl) [inline packages/medusa/src/services/product-variant.js:534]: Change to:
```
@param {Object} config - query config object for variant retrieval
```

**Review by olivermrbl — CHANGES_REQUESTED** [review]
Many thanks for contributing. A couple of changes before we can get it in ❤️

**Review by olivermrbl — APPROVED** [review]
Many thanks for the contribution!! ❤️

**Pull request merged**